### ISSUE TITLE
Increase minimum PHP version

### ISSUE THREAD

**Issue by Gared** [ElephantIO/elephant.io#26]
I would like to increase the minimum PHP version to 7.4 to use newer PHP features.

Regarding packagist the package version 4.* is used with those PHP versions:
* 7.2: 0,5%
* 7.3: 2,3%
* 7.4: 23,8%

If you would agree I could do this update

**Comment by tohenk** [comment]
Just keeps it to 7.2 for now.

**Comment by Gared** [comment]
@tohenk Can you please say why?

**Comment by tohenk** [comment]
It is stated in the readme to support PHP 7.2.

**Comment by Gared** [comment]
Then let's change the README 😅
Or is there any specific reason to keep compatibility with such an old, outdated and non-maintained PHP version?

**Comment by pimjansen** [comment]
> Then let's change the README 😅
> Or is there any specific reason to keep compatibility with such an old, outdated and non-maintained PHP version?

I agree. There is a working version now for the people using it and that will nog change?

Just bump the minimum version to the last supported php version and continue from there? Only just include a proper guide and instruction on this

**Comment by Gared** [comment]
Update:
Regarding packagist the package version 4.* is used with those PHP versions:
* 7.2: 0,2%
* 7.3: 1,8%
* 7.4: 5,5%
* 8.0: 2,2%

**Comment by Gared** [comment]
Can we please increase the minimum PHP version now @tohenk ?
